ee/dtr/admin/configure/deploy-caches/simple-kube.md

```diff
@@ -0,0 +1,229 @@
+
+On your workstation, create a directory called `certs`. Within here place the newly 
+created certificate `cache.cert.pem` and key `cache.key.pem` for your DTR cache. Also
+ place the certificate authority (including any intermedite certificate authorities)
+ of the certificate from your DTR deployment. This could be sourced from curl.
+`curl -s https://<dtr-fqdn>/ca -o certs/dtr.cert.pem`.
+
+### Create the DTR Config
+
+This is the configuration file of the registry component of the DTR Cache. This yaml
+should be updated with the relevant `cache-fqdn` and `dtr-fqdn`.
+
+```
+version: 0.1
+log:
+  level: info
+storage:
+  delete:
+    enabled: true
+  filesystem:
+    rootdirectory: /var/lib/registry
+http:
+  addr: 0.0.0.0:443
+  secret: generate-random-secret
+  host: https://<external-fqdn-dtrcache> # Could be Ingress / LB / Host
+  tls:
+    certificate: /certs/cache.cert.pem
+    key: /certs/cache.key.pem
+middleware:
+  registry:
+      - name: downstream
+        options:
+          blobttl: 24h
+          upstreams:
+            - https://<dtr-url>
+          cas:
+            - /certs/dtr.cert.pem
+```
+
+With this configuration, the cache fetches image layers from DTR and keeps
+a local copy for 24 hours. After that, if a user requests that image layer,
+the cache fetches it again from DTR.
+
+The cache is configured to persist data inside its container.
+If something goes wrong with the cache service, Docker automatically redeploys a
+new container, but previously cached data is not persisted.
+You can [customize the storage parameters](/registry/configuration.md#storage),
+if you want to store the image layers using a persistent storage backend.
+
+### Create Kubernetes Resources
+
+Create a Kubernetes namespace to logical seperate all of our DTR cache components.
+
+```
+$ kubectl create ns dtr
+```
+
+Create the Kubernetes Secrets and the Kubernetes ConfigMaps. Note the commands below
+will only work if your workstation directory was configured our examples structure 
+above. 
+
+```
+$ kubectl create secret generic dtr-certs \
+    --from-file=certs/dtr.cert.pem \
+    --from-file=certs/cache.cert.pem \
+    --from-file=certs/cache.key.pem
+
+$ kubectl create configmap dtr-cache-config \
+    --from-file=config.yml
+```
+
+Create the Kubernetes Deployment. 
+
+```
+cat <<EOF | kubectl create -f -
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: dtr-cache
+  namespace: dtr
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dtr-cache
+  template:
+    metadata:
+      labels:
+        app: dtr-cache
+      annotations:
+       seccomp.security.alpha.kubernetes.io/pod: docker/default
+    spec:
+      containers:
+        - name: dtr-cache
+          image: {{ page.dtr_org }}/{{ page.dtr_repo }}-content-cache:{{ page.dtr_version }}
+          command: ["bin/sh"]
+          args:
+            - start.sh
+            - /config/config.yml
```

Comment:
Should this simply be config.yml?

Based on the directory structure outline at first: 
(there doesn't seem to be a config directory)

```
├── dtrcache.yml        # Yaml file to deploy cache with a single command
├── config.yml          # The cache configuration file
└── certs
    ├── cache.cert.pem  # The cache public key certificate
```